### DISCUSSION TITLE
Welcome to quantlib Discussions!

### DISCUSSION THREAD

**Discussion by ackleymi** [piquette/quantlib#2] [General]
## 👋 Hello!
  We’re using Discussions as a place to coordinate quantlib development progress. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by surfertas** [comment]
Not sure that I can provide much input on the architecture, but happy to help with transporting features.

**Comment by surfertas** [comment]
Will you be creating a dev branch?

**Reply by ackleymi** [reply]
Yes, see https://github.com/piquette/quantlib/tree/develop :)
